content/docs/command-reference/pull.md

```diff
@@ -167,11 +164,11 @@ $ dvc pull train.dvc
 > Please delete the `.dvc/cache` directory first (with `rm -Rf .dvc/cache`) to
 > follow this example if you tried the previous ones.
 
-Our [pipeline](/doc/command-reference/pipeline) has been setup with these
+Our [pipeline](/doc/command-reference/dag) has been setup with these
 [stages](/doc/command-reference/run):
 
 ```dvc
-$ dvc pipeline show evaluate.dvc
+$ dvc dag evaluate.dvc
```

Comment:
The remote isn't working. I cannot find the output of `evaluate.dvc` since it doesn't exist. Also, files in `src/` are Git-tracked files, so I had to remove them from git. Even after adding all files in src/, `dvc dag evaluate.py.dvc` does not show the linked files either. I suppose I am missing something and having the remote changed would result in a better output. FYI, here's my current output for `dvc dag src/evaluate.py.dvc`:
```
+---------------------+  
| src/evaluate.py.dvc |  
+---------------------+  
```